src/lib/swapper/swappers/ThorchainSwapper/utils/addAggregatorAndDestinationToMemo.test.ts

```diff
@@ -0,0 +1,30 @@
+import { describe, expect, it } from 'vitest'
+
+import { addAggregatorAndDestinationToMemo } from './addAggregatorAndDestinationToMemo'
+import { MEMO_PART_DELIMITER } from './constants'
+
+const RECEIVE_ADDRESS = '0x32DBc9Cf9E8FbCebE1e0a2ecF05Ed86Ca3096Cb6'
+const AGGREGATOR_ADDRESS = '0x8a65ac0E23F31979db06Ec62Af62b132a6dF4741'
```

Comment:
Not really a blocker since this is a test, but this isn't an aggregator address (see list here https://gitlab.com/thorchain/thornode/-/blob/develop/x/thorchain/aggregators/dex_mainnet.go) just a regular EOA

Reply:
I took a random address from another test, we are spreading addresses all over the project's tests without sharing them, we might want to introduce a mocked data file somewhere in the future 

Changed it to a real thorchain aggregator address 👍